src/lunch_bot/yelp.clj

```diff
@@ -13,18 +14,37 @@
 (def ^:private zipcode "07302")
 
 (defn- parse-single-result [result]
-  (result :url))
+  (->
+    (str (result :name) " - " (first (first (result :categories))) " - " (result :url))
+    (send/send-response))
+    "done parse-single-result" result)
```

Comment:
Removeing output string causes errors with buffer exiting prematurely. Ex, in curl
```
curl: (18) transfer closed with 20 bytes remaining to read
```